app/dts/bindings/zmk,soft-off-wakeup-sources.yaml

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2023 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: |
+  Description of all possible wakeup-sources from a forces
+  soft-off state.
+
+compatible: "zmk,soft-off-wakeup-sources"
+
+properties:
+  wakeup-sources:
+    type: phandles
+    required: true
+    description: List of wakeup-sources that should be enabled to wake the system from forces soft-off state.
```

```suggestion
    description: List of wakeup-sources that should be enabled to wake the system from soft-off state.
```
extra word? Or was that supposed to be "forced"?

---

Was intended to be "forced". Fixed.